### ISSUE TITLE
aether mod for minetest

### ISSUE THREAD

**Issue by tobyplowy** [22i/aether-voxel-blender-models#1]
@22i maybe it would be cool to Recreate the whole aether demention for minetest?

**Comment by 22i** [comment]
@tobyplowy yes! what do you mean by whole?

**Comment by tobyplowy** [comment]
@22i well I mean it wuld be cool to to recreate the aether mod for minetest (recreate everything)

From mapgen to blocks items and so on :)

**Comment by 22i** [comment]
@tobyplowy  i agree @TechRunner2 started a bit here https://github.com/TechRunner2/aether2 i will help out

**Comment by tobyplowy** [comment]
@22i that project seems abandoned

**Comment by 22i** [comment]
@tobyplowy ye but i can still try to make a pull request and if nothing gets through i can always make a new repository

**Comment by tobyplowy** [comment]
@22i ok that's a good idea,

If i see some progress happening than I will also join to help

**Comment by TechRunner2** [comment]
@22i @tobyplowy 
It has kind of been abandoned because I am just too busy with school and life in general and lack the needed experience with minetest moding to get super far into the project. I would like to continue this project.
